be/test/http/file_cache_action_test.cpp

```diff
@@ -0,0 +1,162 @@
+        _event_base = event_base_new();
+        _evhttp_req = evhttp_request_new(nullptr, nullptr);
+        _action = std::make_unique<FileCacheAction>(nullptr);
+
+        ASSERT_NE(doris::ExecEnv::GetInstance()->file_cache_factory(), nullptr);
+
+        _base_path = "/tmp/file_cache_action_ut_" + std::to_string(getpid());
+        (void)std::filesystem::create_directories(_base_path);
+        doris::io::FileCacheSettings settings;
+        settings.storage = "memory";
+        settings.capacity = 1024 * 1024;          // 1MB
+        settings.max_file_block_size = 64 * 1024; // 64KB
+        auto cache = std::make_unique<doris::io::BlockFileCache>(_base_path, settings);
+        ASSERT_TRUE(cache->initialize());
+        for (int i = 0; i < 1000; ++i) {
+            if (cache->get_async_open_success()) break;
+            std::this_thread::sleep_for(std::chrono::milliseconds(1));
+        }
+        auto* raw = cache.get();
+        auto* factory = doris::io::FileCacheFactory::instance();
+        factory->_caches.emplace_back(std::move(cache));
+        factory->_path_to_cache[_base_path] = raw;
+    }
+
+    void TearDown() override {
+        if (_evhttp_req != nullptr) {
+            evhttp_request_free(_evhttp_req);
+        }
+
+        if (_event_base != nullptr) {
+            event_base_free(_event_base);
+        }
+
+        if (doris::ExecEnv::GetInstance()->file_cache_factory() != nullptr) {
+            doris::io::FileCacheFactory::instance()->clear_file_caches(true);
+        }
+    }
+
+protected:
+    evhttp_request* _evhttp_req = nullptr;
+    event_base* _event_base = nullptr;
+    std::unique_ptr<FileCacheAction> _action;
+    std::string _base_path;
+
+    inline static std::unique_ptr<doris::io::FileCacheFactory> _suite_factory;
+};
+
+TEST_F(FileCacheActionTest, list_base_paths) {
+    HttpRequest req(_evhttp_req);
+    std::string json_metrics;
+
+    req._params["op"] = "list_base_paths";
+
+    Status status = _action->_handle_header(&req, &json_metrics);
+    EXPECT_TRUE(status.ok());
+
+    std::string expected = "[\"" + _base_path + "\"]";
+    EXPECT_EQ(json_metrics, expected);
+}
+
+TEST_F(FileCacheActionTest, check_consistency_missing_param) {
+    HttpRequest req(_evhttp_req);
+    std::string json_metrics;
+
+    req._params["op"] = "check_consistency";
+
+    Status status = _action->_handle_header(&req, &json_metrics);
+
+    EXPECT_FALSE(status.ok());
+    EXPECT_EQ(status.to_string(), "missing parameter: base_path is required");
+}
+
+TEST_F(FileCacheActionTest, check_consistency_not_found) {
+    HttpRequest req(_evhttp_req);
+    std::string json_metrics;
+
+    req._params["op"] = "check_consistency";
+    req._params["base_path"] = "/tmp/file_cache_action_ut_not_exist";
+
+    Status status = _action->_handle_header(&req, &json_metrics);
+
+    EXPECT_FALSE(status.ok());
+    EXPECT_EQ(status.to_string(),
+              "file cache not found for base_path: /tmp/file_cache_action_ut_not_exist");
```

Comment:
[INVALID_ARGUMENT]file cache not found for base_path: /tmp/file_cache_action_ut_not_exist